### PR TITLE
Add Gitpod hackathon guide

### PR DESCRIPTION
**Pull request by maksim36ua**
Blog post about Gitpod + Wasp integration and how to use it during the hackathon

### CODE REVIEW

File: web/blog/2022-06-01-gitpod-hackathon-guide.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+title: How to win a hackathon. Brief manual.
+authors: [maksym36ua]
+image: /img/martijn-intro-banner.png
+tags: [wasp, gitpod, hackathon]
+---
+
```
Comment (shayneczyzewski) [inline web/blog/2022-06-01-gitpod-hackathon-guide.md:7]: I think for most posts @matijaSos recommends we use:

```
import InBlogCta from './components/InBlogCta';
import WaspIntro from './_wasp-intro.md';

<WaspIntro />
<InBlogCta />
```

May want to here as well for consistency.

File: web/blog/2022-06-01-gitpod-hackathon-guide.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+>*"All good thoughts and ideas mean nothing without the proper tools to achieve them."*<br/>
+>*Jason Statham*
+
+
+
+**TL;DR:** Wasp allows you to build and deploy a full-stack JS web app with a single config file. Gitpod spins up fresh, automated developer environments in the cloud, in seconds. A perfect tandem to win a hackathon and enjoy free pizza even before other teams even started to set up their coding env and realized they need to update their node version.
```
Comment (shayneczyzewski) [inline web/blog/2022-06-01-gitpod-hackathon-guide.md:14]: Perhaps we should link to Gitpod here or the reference below?

**Review by shayneczyzewski — COMMENTED** [review]
Looking great @mkhamrov! I left a comment about the standard intro and call to action you may want to consider.

Additionally, perhaps we should link to Gitpod on your first reference?

Otherwise, looking good to me. We can let @matijaSos do the final 👍🏻 while catching up next week (after the wedding is over this weekend). Thanks and great job!

File: web/blog/2022-06-01-gitpod-hackathon-guide.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
+---
+title: How to win a hackathon. Brief manual.
+authors: [maksym36ua]
+image: /img/martijn-intro-banner.png
```
Comment (Martinsos) [inline web/blog/2022-06-01-gitpod-hackathon-guide.md:4]: This one I believe you will want to replace for your own banner?

File: web/blog/2022-06-01-gitpod-hackathon-guide.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+Since Gitpod is a cloud-based workspace – spinning up a new application takes a couple of clicks. 
+
+1) Fork https://github.com/gitpod-io/template-wasp and give it a meaningful name, e.g. “My Awesome Recipes App” -> this is now a repo for your new web app.<br/>
+2) In your newly created repo, check the Readme and click the “Open in Gitpod” button<br/>
+3) Login via Github<br/>
+4) Allow pop-ups<br/>
+5) That’s it! Enjoy your fresh cloud-based dev environment!<br/>
+
+![Pennywise luring to take part in hackathon](../static/img/gitpod-roadmap.png)
```
Comment (Martinsos) [inline web/blog/2022-06-01-gitpod-hackathon-guide.md:45]: Love the description :D

File: web/blog/2022-06-01-gitpod-hackathon-guide.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+An optional thing might be enabling the “Share” option to make the app accessible from the external internet. 
+
+![How to share a workspace](../static/img/gitpod-share.jpg)
+
+You can pick up one of the following IDE’s, switch between light/dark themes and you can even install all your favorite extensions. 
+
+![How to share a workspace](../static/img/gitpod-ide.jpg)
```
Comment (Martinsos) [inline web/blog/2022-06-01-gitpod-hackathon-guide.md:53]: Wrong description?

**Review by Martinsos — COMMENTED** [review]
Looking great!!! Left a tiny comment or two, will let @matijaSos do the final approving. Yay first PR :D!

File: web/blog/2022-06-01-gitpod-hackathon-guide.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
+---
+title: How to win a hackathon. Brief manual.
+authors: [maksym36ua]
+tags: [wasp, gitpod, hackathon]
```
Comment (matijaSos) [inline web/blog/2022-06-01-gitpod-hackathon-guide.md:4]: we have no title image? shall we add something, pennywise one maybe?

Comment (maksim36ua) [reply web/blog/2022-06-01-gitpod-hackathon-guide.md:4]: I'll come up with a pretty photo when I'll be able to find the initial image I'm using right now :)

**Pull request merged**